app/middleware/grpc/client.js

```diff
@@ -14,18 +14,16 @@ var Buffer = require('buffer/').Buffer;
 export function client(address, port, cb) {
     var protoDescriptor = grpc.load('./app/api.proto');
     var walletrpc = protoDescriptor.walletrpc;
-    /*
     var certPath = path.join(process.env.HOME, '.dcrwallet', 'rpc.cert');
```

Comment:
I think this will throw an exception if HOME is not a defined environment variable (e.g. windows).  It should be moved into a final `else` branch below.

I should also fix this in the dcrwallet doco.